src/ApiPort/ApiPort.GUI/README.md

```diff
@@ -1,6 +1,6 @@
-## Portability Analyzer 
+## .NET Portability Analyzer 
 
-The Portability Analyzer is an open source standalone tool that simplifies your porting experience by identifying APIs that are not portable to .NET Core. We discovered that developers are struggling to port their desktop applications from .NET Framework to .NET Core, therefore in order to meet customer needs we updated the .NET Portability Analyzer. The Portability Analyzer is an open source tools that simplifies your porting experience by identifying APIs that are not portable among the various .NET Platforms. 
+The .NET Portability Analyzer is an open source standalone tool that simplifies your porting experience by identifying APIs that are not portable to .NET Core. We discovered that developers are struggling to port their desktop applications from .NET Framework to .NET Core, therefore in order to meet customer needs we updated the .NET Portability Analyzer. The Portability Analyzer is an open source tools that simplifies your porting experience by identifying APIs that are not portable among the various .NET Platforms. 
```

Comment:
"...that simplifies your porting experience..." what porting experience? From Framework 4.5 to Framework 4.6? From Framework to Core? From one version of .NET to another? Also, does it only show APIs that aren't portable to .NET Core, or compatibility within different framework versions? Or .NET Standard portability as well? :)

You could also point to [this](https://github.com/microsoft/dotnet-apiport/tree/dev/docs/HowTo) link as a more info reference. 

Also, 'We discovered that developers are struggling to port their..." - normal developers looking at this probably don't care about this. This is a motivator for the GUI tool. 

"The Portability Analyzer is an open source tool~s~ that simplifies..."